tools/packaging/kernel/configs/powerpc_kata_kvm_5.4.x

```diff
@@ -757,7 +757,7 @@ CONFIG_INET_TCP_DIAG=y
 CONFIG_TCP_CONG_CUBIC=y
 CONFIG_DEFAULT_TCP_CONG="cubic"
 # CONFIG_TCP_MD5SIG is not set
-# CONFIG_IPV6 is not set
+CONFIG_IPV6=y
```

Comment:
I'd prefer the kernel changes in a different patch.  This can be part of the same series, but in a different patch, if possible.

Reply:
Thanks @fidencio!  I will make a different PR. Shall a new issue be for it ? or just using the same issue #1333?

Reply:
See https://github.com/kata-containers/kata-containers/pull/1314#issuecomment-769960866